rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -45,9 +45,11 @@ typedef struct BoundingBoxCord_
 typedef  struct { float xc; float yc; float w; float h; } BoundingBoxCord_xcycwh;
 typedef  std::vector<BoundingBoxCord> BoundingBoxCords;
 typedef  std::vector<BoundingBoxCord_xcycwh> BoundingBoxCords_xcycwh;
-typedef  std::vector<int> BoundingBoxLabels;
+typedef  std::vector<int> Labels;
 typedef  struct { int w; int h; } ImgSize;
 typedef  std::vector<ImgSize> ImgSizes;
+typedef std::vector<std::vector<float>> coords;
```

Comment:
Lets keep it as Coords, as all other variables name starts with CAPS letter to maintain uniformity

Reply:
It is not used anywhere. So removing it